app/models/evmbaseline.rb

```diff
@@ -17,4 +21,15 @@ def bac
     evmbaselineIssues.sum(:estimated_hours).round(1)
   end
 
+  acts_as_event :title => Proc.new { |o| l(:title_evm_tab) + ':' + o.subject },
+                :description => l(:label_ativity_message_new),
+                :datetime => :created_on,
+                :type => Proc.new { |o| 'EvmBaseine-' + (o.created_on < o.updated_on ? 'edit' : 'new') },
+                :url => Proc.new { |o| {:controller => 'evmbaselines', :action => 'index', :project_id => o.project} }
+
+  acts_as_activity_provider :scope => joins(:project),
+                            :permission => :view_evm_baselines,
+                            :type => 'evmbaseline',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.